tests/run-tests.sh

```diff
@@ -41,13 +47,13 @@ check_exists() {
 
 # check exit code
 check_exit_code() {
-if [ $? -eq 0 ]
-then
-    echo "[OK] : $1"
-else
-    echo "*** Error - $2!" >&2
-    exit -1
-fi
+    if [ $? -eq 0 ]
```

Comment:
whitespace?

Reply:
It is inside the function body, hence I indented it for readibility.

Reply:
looks like tabs and not spaces.

Reply:
I rechecked, they are spaces.